tests/static_map/erase_test.cu

```diff
@@ -32,51 +30,49 @@ using size_type = int32_t;
 template <typename Map>
 void test_erase(Map& map, size_type num_keys)
 {
-  using Key   = typename Map::key_type;
-  using Value = typename Map::mapped_type;
+  using key_type    = typename Map::key_type;
+  using mapped_type = typename Map::mapped_type;
 
-  thrust::device_vector<Key> d_keys(num_keys);
-  thrust::device_vector<Value> d_values(num_keys);
   thrust::device_vector<bool> d_keys_exist(num_keys);
 
-  thrust::sequence(thrust::device, d_keys.begin(), d_keys.end(), 1);
-  thrust::sequence(thrust::device, d_values.begin(), d_values.end(), 1);
+  auto keys_begin = thrust::counting_iterator<key_type>(1);
 
-  auto pairs_begin =
-    thrust::make_zip_iterator(thrust::make_tuple(d_keys.begin(), d_values.begin()));
+  auto pairs_begin = thrust::make_transform_iterator(keys_begin, [] __device__(key_type const& x) {
+    return cuco::pair<key_type, mapped_type>(x, static_cast<mapped_type>(x));
+  });
```

Comment:
guard device lambda with `cuda::proclaim_return_type`

Reply:
Addressed in [6cc9040](https://github.com/NVIDIA/cuCollections/pull/387/commits/6cc9040acc17e1873d5b3976ac07438a7da3b644)